From 274845e693458df107093842ea3d4063e0a1d67e Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Wed, 10 Jul 2024 09:22:36 +0200
Subject: [PATCH 1195/1695] plat-stm32mp1: pm: correct header file inclusion
 order

Correct the order of the header file inclusion in context.h.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 60375a8e99ea1a2a8d62141889490f00f6d504f2 plat-stm32mp1: pm: correct header file inclusion order]
Change-Id: Ic5e2c56a5d8963cd2f2c3fcd0b8551b8b19a5d53
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/392463
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp1/pm/context.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/pm/context.c b/core/arch/arm/plat-stm32mp1/pm/context.c
index 878a40a05..6ea8cd941 100644
--- a/core/arch/arm/plat-stm32mp1/pm/context.c
+++ b/core/arch/arm/plat-stm32mp1/pm/context.c
@@ -16,9 +16,9 @@
 #include <generated/context_asm_defines.h>
 #include <initcall.h>
 #include <io.h>
+#include <keep.h>
 #include <kernel/cache_helpers.h>
 #include <kernel/delay.h>
-#include <keep.h>
 #include <kernel/panic.h>
 #include <kernel/pm.h>
 #include <mm/core_memprot.h>
-- 
2.47.2

